release.mk

```diff
@@ -457,8 +460,10 @@ ifeq ($(THIRD_PARTY_SOFTWARE_LOCATION),)
 	$(error THIRD_PARTY_SOFTWARE_LOCATION must be set to do copy-external)
 endif
 
-	# Copy IDE plugin
+ifeq ($(ENABLE_IDE_PLUGINS_BUILD),)
```

Comment:
Check should be `ifneq ($(ENABLE_IDE_PLUGINS_BUILD),y)` so `y` is the only value that means "yes" and any other value means "no".